frame/staking/src/benchmarking.rs

```diff
@@ -407,35 +407,55 @@ benchmarks! {
 	}
 
 	submit_solution_initial {
-		let v in 1 .. 10;
-		let n in 1 .. 100;
+		// NOTE: this benchmark requires: `v > w`, `n > a` and `a > w`. Talk to @kianenigma if you
+		// want to know why.
+		// number of validator intentions
+		let v in 50 .. 100;
```

Comment:
FYI there is 367 validators intentions in kusama (but should be fine because it should be linear anyway)

Reply:
yeah I otherwise the execution time of tests and benchmarks will bloat :( I tried it once with higher value but it is fine and linear.